shared/message/skill/Look.proto

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of the NUbots Codebase.
+ *
+ * The NUbots Codebase is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * The NUbots Codebase is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with the NUbots Codebase.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright 2023 NUbots <nubots@nubots.net>
+ */
+syntax = "proto3";
+
+package message.skill;
+import "Vector.proto";
+
+/// A message telling the robot to look in the given direction
+message Look {
```

Comment:
I think `LookAt` is more prescriptive as a task, and it would avoid name collisions in your module

Reply:
But we're not telling it what to look at, we're telling it where to look